src/components/Profile.tsx

```diff
@@ -0,0 +1,39 @@
+import Navbar from '@/components/Navbar';
+import { AuthProps } from '@/types/types';
+import { PassageUser } from '@passageidentity/passage-elements/passage-user';
+import dotenv from 'dotenv';
+import Router from "next/router";
+import { useEffect } from 'react';
+
+function Profile({isAuthorized, userID}: AuthProps) {
+  useEffect(() => {
+    require("@passageidentity/passage-elements/passage-profile");
+    if (!isAuthorized) {
+      Router.push("/"); 
+    }
+  },[isAuthorized]);
+
+  const signOut = async () => {
+    if (userID !== 'ABrrCENR3M0I6XZ7NLA7gNCY'){
+      new PassageUser().signOut();
+    }
+    Router.push("/");
+  };
+
+  dotenv.config();
+  const passageAppId = process.env.NEXT_PUBLIC_PASSAGE_APP_ID;
+
+  if (!passageAppId) {
+    throw new Error('NEXT_PUBLIC_PASSAGE_APP_ID is not defined in the environment.');
+  }
+
+  return (
+    <div className='flex flex-col content-center'>
+      <h1 className='m-7 text-center text-3xl'>{(userID === 'ABrrCENR3M0I6XZ7NLA7gNCY') && 'Log in to enjoy our profile feature!'}</h1>
+      <passage-profile app-id={process.env.NEXT_PUBLIC_PASSAGE_APP_ID}></passage-profile>
```

Comment:
I love this!! I was worried that the users would have access to edit the demo users data but i love that you made it so that they can't do that!